frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx

```diff
@@ -163,6 +189,7 @@ export const VMDisks: React.FC<VMDisksProps> = ({ vmLikeEntity, pvcs, datavolume
 
 export const VMDisksFirehose: React.FC<VMLikeEntityTabProps> = ({ obj: vmLikeEntity }) => {
   const namespace = getNamespace(vmLikeEntity);
+  const vmTemplate = getVMTemplateNamespacedName(vmLikeEntity);
```

Comment:
can you please refactor this function so it uses `LABEL_USED_TEMPLATE_NAME` and `LABEL_USED_TEMPLATE_NAMESPACE` constants

Reply:
it uses these constants. am I missing something ?

Reply:
I already posted a PR which fixed that